src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -18,10 +18,7 @@ package com.squareup.kotlinpoet
 import com.google.common.collect.ImmutableMap
 import com.google.common.truth.Truth.assertThat
 import com.google.testing.compile.CompilationRule
-import com.squareup.kotlinpoet.KModifier.ABSTRACT
-import com.squareup.kotlinpoet.KModifier.INTERNAL
-import com.squareup.kotlinpoet.KModifier.PRIVATE
-import com.squareup.kotlinpoet.KModifier.VARARG
+import com.squareup.kotlinpoet.KModifier.*
```

Comment:
Jetbrains being helpful. Should probably keep my settings in a repository now with all the different ones I use - tough to keep track